### PR TITLE
by friday22

### PR DESCRIPTION
**Pull request by friday22**
i have added a line at bottom of the file.

### CODE REVIEW

File: evolvo.txt [hunk 1]
```diff
@@ -1,4 +1,7 @@
+
 Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
 Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
 It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. 
 It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+
+created for educational purpose.
```
Comment (ForeverKinghts) [inline evolvo.txt:7]: This line seems very useful

File: evolvo.txt [hunk 2]
```diff
@@ -1,4 +1,7 @@
+
 Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
 Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
 It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
```
Comment (ForeverKinghts) [inline evolvo.txt:4]: this line seems useless....

Comment (friday22) [reply evolvo.txt:4]: okay

Comment (ForeverKinghts) [inline evolvo.txt:4]: new comment on same line

**Review by ForeverKinghts — APPROVED** [review]

**Pull request merged**